examples/csharp/App/Program.cs

```diff
@@ -14,36 +19,162 @@ namespace App
     class Program
     {
         // Replace this with the actual issuer ID you've got from Wallester
-        private static string _issuer = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx";
+        private const string Issuer = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx";
 
         // Replace this with the actual audience ID you've got from Wallester
-        private static string _audience = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx";
+        private const string Audience = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX";
 
         // Replace this with actual Wallester API URL
-        private static string _apiURL = "https://xxx.wallester.eu/v1/test/ping";
+        private const string ApiUrl = "https://xxx.wallester.eu";
+        private const string PingPath = "/v1/test/ping";
+        private const string UploadKycDocumentPath = "/v1/kyc-documents";
+
+        private const string Subject = "api-request";
+
+        // Replace this with actual data to send upload kyc documents request
+        private const string AuditSourceType = "Backend";
+        private const string AuditUserId = "integration-example";
+        private const string ProductCode = "PRODUCTCODE";
+        private const string KycCheckId = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx";
+        private const string FilePath = "../../testdata/file_name.jpg";
+
+        #if READ_KEYS_FROM_STRINGS
+            private const string PrivateKeyPem = """
+            -----BEGIN RSA PRIVATE KEY-----
+            ...
+            ...
+            ...
+            -----END RSA PRIVATE KEY-----
+            """;
 
-        private static string _subject = "api-request";
+            private const string PublicKeyPem = """
+            -----BEGIN PUBLIC KEY-----
+            ...
+            ...
+            ...
+            -----END PUBLIC KEY-----
+            """;
+        #endif
 
         public static void Main(string[] args)
         {
-            var signingCredentials = ReadSigningCredentials("../../keys/example_private.pkcs12", "123456");
-            var wallesterPublicKey = ReadPublicKey("../../keys/example_wallester_public.cer");
+            #if READ_KEYS_FROM_STRINGS
+                var signingCredentials = ReadSigningCredentialsFromString(PrivateKeyPem);
+                var wallesterPublicKey = ReadPublicKeyFromString(PublicKeyPem);
+            #else

```

Comment:
I don't know whether to remove this part or leave it. I used this because it's easier to test locally

Reply:
lets keep it then :)